Cloudinary-pwa-kit/packages/pwa-cloudinary/app/components/cloudinary-lineItem-image/index.jsx

```diff
@@ -11,16 +13,22 @@ const CloudinaryLineItemImage = ({ cldProduct = {}, image = {} }) => {
             {cldProduct?.miniCartImage?.url ? (
                 <Box w="24" flex="none">
                     <AspectRatio ratio="1">
-                        <img src={cldProduct?.miniCartImage?.url} alt={image && image.alt ? image.alt : null} />
+                        <Image className={cldProduct?.miniCartImage?.isResponsive && 'cld-responsive'}
+                            src={cldProduct?.miniCartImage?.url.lastIndexOf('?') > -1 ? cldProduct?.miniCartImage?.url.substring(0, cldProduct?.miniCartImage?.url.lastIndexOf('?')) + cloudinary.CLD_TRACKING_PARAM : cldProduct?.miniCartImage?.url + cloudinary.CLD_TRACKING_PARAM} alt={image && image.alt ? image.alt : null}
+                            srcset={!cldProduct?.miniCartImage?.isResponsive && cldProduct?.miniCartImage?.srcset && updateTrackingParam(cldProduct?.miniCartImage?.srcset)}
+                            sizes={!cldProduct?.miniCartImage?.isResponsive && cldProduct?.miniCartImage?.sizes}
+                        />
                     </AspectRatio>
                 </Box>
             ) : (
                 <>
                     {cldProduct?.url && (
                         <Image
+                            className={cldProduct?.isResponsive && 'cld-responsive'}
                             alt={image.alt}
-                            src={`${cldProduct.url}`}
-                            ignoreFallback={true}
```

Comment:
why was this line removed? (I don't know if it is needed)

Reply:
Yeah, its not needed.